test/functional/feature_poolswap.py

```diff
@@ -537,14 +529,10 @@ def update_comission_and_fee_to_1pct_pool2(self):
         assert_equal(attributes['v0/live/economy/dex/%s/fee_burn_a'%(self.idBL)], Decimal(str(round(dexoutfee, 8))))
 
     def revert_to_initial_state(self):
```

Comment:
Why is something like back to initial state being rolled back 30 blocks? Avoid calling key helpers inaccurately only for a specific context - better to rename to something more accurate or back to reset the node to it's initial position by restarting it.